android/app/src/main/java/com/zulipmobile/notifications/FCMPushNotifications.kt

```diff
@@ -33,6 +37,8 @@ val ACTION_CLEAR = "ACTION_CLEAR"
 val EXTRA_NOTIFICATION_DATA = "data"
 
 fun createNotificationChannel(context: Context) {
+    val audioAttr: AudioAttributes = AudioAttributes.Builder()
+    .setUsage(AudioAttributes.USAGE_NOTIFICATION).build()
```

Comment:
nit: second line should be indented

(really by whatever the autoformatter wants to do, probably, but if doing it without an autoformatter I'd go for 4 spaces)